### ISSUE TITLE
Confd toml configuration should use prefix

### ISSUE THREAD

**Issue by frostynova** [Metaswitch/calico-docker#137]
Since etcd may be used by others (such as swarm), confd should use prefix to limit the impact when etcd key changed, for example:

```
[template]
src = "bird.cfg.template"
dest = "/config/bird.cfg"
prefix = "/calico/v1"
keys = [
    "/host",
    "/ipam/v4/pool",
    "/config"
]
reload_cmd = "pkill -HUP bird || true"
```

I think this is one reason of my problem: https://github.com/Metaswitch/calico-docker/pull/125#issuecomment-102298014
Another is https://github.com/Metaswitch/calico-docker/issues/136

**Comment by robbrockbank** [comment]
@frostynova - good spot - I'm working on adding that in.  i'll update as soon as it is in master.

**Comment by robbrockbank** [comment]
@frostynova - this change has been made and is now checked in to the latest master.  This will be included in the next release.

**Comment by robbrockbank** [comment]
Fixed by #141
